docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -120,6 +120,13 @@ If you're using GitHub Action to conduct `deployments`, please select "Detect De
 - Deployment: A GitHub Action job with a name that matches the given regEx will be considered as a deployment.
 - Production: A GitHub Action job with a name that matches the given regEx will be considered a job in the production environment.
 
+A GitHub workflow run has many jobs. Each GitHub workflow run is converted to a 
```

Comment:
"A GitHub workflow run has many jobs" -> "A GitHub workflow has many jobs" (remove the "run")

Reply:
But run is an execution, workflow is just a definition? So workflow run is converted to cicd_pipeline. Will it be ambiguous if we remove run?